pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+set -ex
+refspec="$1"
+ZUUL_CHANGES="$2"
+keep_env="$3"
+
+FETCH_URL=http://10.71.212.50/p/
```

Comment:
You can set a default value but make this as an input parameter which will be passed downed from zuul/jenkins when it's invoked from jenkins jobs.

Reply:
OK, will add it